.github/workflows/npm-checks.yml

```diff
@@ -1,24 +0,0 @@
-name: NPM Checks
-
-on:
-  pull_request:
-    branches: [master, shutdown-master]
```

Comment:
I don't see a `master` branch here, just `main`, and I am not sure about this `shutdown-master` value - I omitted it in the pr-checks.yml file for now, but I can add it back if needed.

Reply:
This must have been my mistake. `master` is obviously the branch that this workflow targeted back in the `developer-portal` repo and I didn't update it when it moved to this repo. So this file never really did anything. 🤦‍♂️

Reply:
`shutdown-master` is a thing that I setup on `developer-portal` where, even when we were in a government shutdown, we could still have a place to use as a temporary master branch, with all the PR checking completed, but it wouldn't trigger a new MR as MRs during a shutdown are severely limited to break-fix deployments.